maintainers/maintainer-list.nix

```diff
@@ -3169,6 +3169,12 @@
     githubId = 4458;
     name = "Ivan Kozik";
   };
+  ivan-timokhin = {
+    email = "nixpkgs@ivan.timokhin.name";
+    name = "Ivan Timokhin";
+    github = "ivan-timokhin";
+    githubId = 9802104;
+  };
```

Comment:
This should be two commits:

```
maintainers: add ivan-timokhin
```

and

```
junicode: add ivan-timokhin to maintainers
```
You can change this with an interactive rebase (`git rebase -i`, mark this commit as edit, split it into two commits). If you need more help with this, feel free to ask.

Reply:
Like this?